src/main/resources/application.properties

```diff
@@ -5,4 +5,4 @@ spring.datasource.password=kodilla_password
 spring.jpa.database=mysql
 spring.jpa.show-sql=true
 spring.jpa.hibernate.ddl-auto=update
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5InnoDBDialect
\ No newline at end of file
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
```

Comment:
Dlaczego chcesz to zmienić?

Reply:
Poprawiłem na MySQL5InnoDBDialect. Zgłębiłem temat jakie są różnice pomiędzy tymi dwoma. Nie chcę zmieniać. Nie dopatrzyłem w application properties.